Transportation/Bike/BikeHireDockingStation/harvest/bicycle_hire_station_barcelona_harvest.py

```diff
@@ -0,0 +1,169 @@
+#!bin/python
+# -*- coding: utf-8 -*-
+
+"""
+Harmonises data from the city of Barcelona corresponding to
+the bicycle hiring stations
+"""
+
+from __future__ import print_function
+import urllib2
```

Comment:
See: https://stackoverflow.com/questions/2792650/python3-error-import-error-no-module-name-urllib2
```python
try:                 # Python 3
    from urllib.request import Request, urlopen
    from urllib.error import URLError
except ImportError:  # Python 2
    from urllib2 import Request, URLError, urlopen
```
Yes, it is ugly but it is correct for compatibility with both Python 2 and Python 3.  The alternative is to add a third party dependency such as [requests](http://docs.python-requests.org) or [six](https://pythonhosted.org/six).